Src/GoogleApis.Auth/OAuth2/ServiceCredential.cs

```diff
@@ -123,7 +123,7 @@ public TokenResponse Token
                     return token;
                 }
             }
-            protected set
+            set

```

Comment:
I had added this to support the GRPC case of setting tokens. Removing this for now, if this causes concern. This can be added subsequently.